Code/Tools/TestImpactFramework/CMakeLists.txt

```diff
@@ -14,11 +14,18 @@ if(PAL_TRAIT_TEST_IMPACT_FRAMEWORK_SUPPORTED)
     add_subdirectory(Runtime)
     add_subdirectory(Frontend)
     ly_add_pytest(
-        NAME TiafToolsTest
-        PATH ${LY_ROOT_FOLDER}/scripts/build/TestImpactAnalysis/test_tiaf_tools.py
+        NAME TIAFToolsTest
+        PATH ${LY_ROOT_FOLDER}/scripts/build/TestImpactAnalysis/Testing/test_tiaf_tools.py
         TEST_SERIAL
         COMPONENT 
             TestImpactFramework
         
     )
+    ly_add_pytest(
+        NAME TIAFPythonUnitTest
```

Comment:
nit: Same with the naming here

Reply:
See https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#abbreviations for more info